.github/workflows/main.yml

```diff
@@ -48,50 +48,46 @@ jobs:
         with:
           python-version: "3.11"
       
-      - name: Install gha-token
-        run: python3.11 -m pip install "git+https://github.com/hypothesis/gha-token.git"
-      
-      - name: Get GitHub token
-        id: github_token
-        run: echo GITHUB_TOKEN=$(gha-token --app-id "$APP_ID" --installation-id "$INSTALL_ID" --private-key "$PRIVATE_KEY") >> $GITHUB_OUTPUT
-        env:
-          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
-          APP_ID: ${{ secrets.APP_ID }}
-          INSTALL_ID: ${{ secrets.INSTALL_ID }}
+      - name: Get GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
       
       - name: Checkout repo
         uses: actions/checkout@v3
         with:
           repository: "washabstract/artemis"
-          token: ${{ steps.github_token.outputs.GITHUB_TOKEN }}
-      
+          token: ${{ steps.app-token.outputs.token }}
+
       - name: Update .env file
-        run: echo "IMAGE_TAG=${GITHUB_SHA}" > aws/.env
+        run: sed -i '/^CYCLADES_IMAGE_TAG=/d' aws/.env && echo "CYCLADES_IMAGE_TAG=${GITHUB_SHA}" >> aws/.env
```

Comment:
this works for now but I'm a little concerned about conflicts arising. hopefully this won't be too much of an issue given athens commits are going to drop but if it is we can revisit our approach